elements/pfe-styles/package.json

```
@@ -1,11 +1,13 @@
 {
-  "name": "@patternfly/pfe-layouts",
+  "name": "@patternfly/pfe-styles",
   "pfelement": {
     "className": "PfeLayouts",
     "elementName": "pfe-layouts",
     "assets": [
       "pfe-base.*css",
       "pfe-base.*.map",
+      "pfe-context.*css",
```

Colors files are missing:
```suggestion
      "pfe-colors.*css",
      "pfe-colors.*.map",
      "pfe-context.*css",
```

---

We're not shipping the color file - that's just for use in the demo page as a preview tool so it shouldn't go in the package.json.